.github/workflows/linux.yml

```diff
@@ -168,8 +174,9 @@ jobs:
         cmake ..                        \
             -DCMAKE_BUILD_TYPE=Debug    \
             -DCMAKE_VERBOSE_MAKEFILE=ON \
-            -DAMReX_ENABLE_TESTS=ON    \
-            -DAMReX_PARTICLES=ON       \
+            -DAMReX_ENABLE_TESTS=ON     \
+            -DAMReX_FORTRAN=OFF         \
```

Comment:
This is the intentional w/o Fortran test. I am adding the flag just to be explicit here, even though it is the default.